treeherder/webapp/api/classifiedfailure.py

```diff
@@ -1,6 +1,6 @@
 from collections import defaultdict
 
-import rest_framework_filters as filters
+import django_filters
```

Comment:
yeah, its confusing how many class share the same name in drf and django.
So should I go ahead and make the changes in other files?